tensorflow/examples/label_image/label_image.py

```diff
@@ -0,0 +1,125 @@
+# Copyright 2017 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
```

Comment:
Thanks for the review.

This is a python implementation that provides code corresponding to the C++ code. With that, we have more intuitive mapping between C++ and Python than the python code mentioned in the [Inception tutorial](https://tensorflow.org/tutorials/image_recognition/) and could be easier to add visualization or debug code.

I pushed the patch with README.md is updated.